geomstats/backend/tensorflow.py

```diff
@@ -5,6 +5,10 @@
 import tensorflow as tf
 
 
+def tile(*args, **kwargs):
```

Comment:
can we avoid passing args and kwargs in the tensorflow backend? I've been passing converting the numpy arguments explicitely so far.